src/P.hs

```diff
@@ -54,6 +54,11 @@ import           Data.Bitraversable as X (
                    , bisequenceA
                    , bifor
                    )
+import           Data.Bifoldable as X (
+                     Bifoldable (..)
+                   , bisequence_
+                   , bifor_
+                   )
```

Comment:
I think this will be missing [bitraverse_](http://hackage.haskell.org/package/bifunctors-5.4.1/docs/Data-Bifoldable.html#v:bitraverse_)